scripts/globals/mobskills/Numbing_Noise.lua

```diff
@@ -12,6 +12,19 @@ require("/scripts/globals/status");
 require("/scripts/globals/monstertpmoves");
 
 ---------------------------------------------
+-- For Race Runner
+local path = {
```

Comment:
yep, if you need anything extra in the call (like the mobskill itself) i'll add it for you